tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -257,4 +257,52 @@ public function testOnlyOwnCommentIsParsed(): void
             ->getColumn('col1')
             ->getComment());
     }
+
+    public function testUnnamedForeignKeyConstraintHandling(): void
+    {
+        $sql = <<<SQL
+CREATE TABLE "bug4243_types" (
```

Comment:
Would it be possible to not have references to the bug in the test? The reference is available through Git history. The uniqueness of the table names needs to be guaranteed within one test file.

It would be even better if the tables were named according to their roles in the test (e.g. `referencing`, `referenced`).

Reply:
the connection is reused accross the whole test setup based on `DbalFunctionalTestCase`, which would lead to potential clashes in table names. The namespacing of the table name is necessary or not?

Reply:
Actually, I was wrong (confused with something else). The tests are responsible for the uniqueness of their tables. For instance:

https://github.com/doctrine/dbal/blob/3f3551aadb20c7ed58ee9c7f06ed74770588272f/tests/Doctrine/Tests/DBAL/Functional/TemporaryTableTest.php#L15-L18

In this case, adding `DROP TABLE IF EXISTS` in the beginning of the script will suffice.